src/bfs.h

```diff
@@ -0,0 +1,52 @@
+/* Breadth First Search Implementation using Adjacency List 
+ *	representation of the graph
+ *
+ * Example:
+ *	#include "bfs.h"
+ *
+ *	int main() {
+ *		struct Graph *G;
+ *		struct attributes *n;
```

Comment:
Okay.
I wanted to make sure that we are mallocing in Cython.
Also, I think I need to remove the `malloc` in `bfs()`.